build/build.ps1

```diff
@@ -44,11 +44,13 @@ function restore() {
 }
 
 function deploy([string] $version) {
-    #DeployOnRepox $scannerZipPath "" $version
-    $scannerZipPath = Get-Item .\DeploymentArtifacts\BuildAgentPayload\Release\SonarQube.Scanner.MSBuild.zip
+    #DeployOnRepox $classicScannerZipPath "" $version
+    $classicScannerZipPath = Get-Item .\DeploymentArtifacts\BuildAgentPayload\Release\SonarQube.Scanner.MSBuild-net46.zip
```

Comment:
[cosmetic] I don't like this mix of styles, I'd rather have only one style. You are using `SonarQube.Scanner.MSBuild` and then `-net46` or `-netcoreapp2.0`.

For me it should be `SonarQube.Scanner.MSBuild.Net46` and `SonarQube.Scanner.MSBuild.NetCoreApp20` or (but this might break other things) `sonarqube-scanner-msbuild-net46` and `sonarqube-scanner-msbuild-netcoreapp20`.

Reply:
Also I think we should change the name of the extension from `SonarQube` to `SonarSource` because this tool can also be used to target `SonarCloud` so the naming is confusing. Not sure whether this change should be done now or later.

Reply:
@Evangelink be careful with such a change. It should be decided with great care. SonarSource is not the brand of a product. Looking at what was done on VSTS, the trend is instead to have dedicated tools for SonarCloud (so we would have a SonarCloud Scanner for MSBuild).

Reply:
It's not urgent that we do it now, and I think we should discuss it with the PMs and Support first.

Reply:
@henryju I agree that the rename should be done carefully. For the time being there is no need in having a separate tool for _SonarCloud_ as the 2 tools are the same so maybe something like `Sonar.Scanner.MSBuild` would be better.

Reply:
I agree with both of your comments yet my first comment should be addressed IMO.

Reply:
`SonarQube` is also used in names of assemblies, so it would be confusing. I like the other suggestion. What do you think about `sonarqube-scanner-msbuild-(target framework)`? The "lowercase dash" format is used in repox, on github releases and in the scanner documentation already.

Reply:
It was my second suggestion but was actually my favorite (I would just remove the `.` from `netcoreapp2.0`).

Reply:
Not a huge fan of the dot either, but that way it's consistent with TFM: https://docs.microsoft.com/en-us/dotnet/standard/frameworks